### DISCUSSION TITLE
Does more(1) support other monitor technologies than cathode-ray tube?

### DISCUSSION THREAD

**Discussion by eepp** [util-linux/util-linux#1891] [Q&A] [answered]
The more(1) manual page says:

> more - file perusal filter for crt viewing

Could I use the `more` command with, say, a plasma display panel, or even a liquid-crystal display?

**Comment by karelzak** [comment, marked as answer]
No, you have to buy and use CRT to get the right feeling from more(1) :-)

https://github.com/util-linux/util-linux/commit/ed07532a97fbfe01ef2b2d855280ca205f850b3c
